packages/vats/README.md

```diff
@@ -0,0 +1,3 @@
+# Vats
+
+Builders for Vats on the Agoric chain that aren't contracts.
```

Comment:
there are some contracts here too; `mintHolder` and `centralSupply` come to mind.
But this will do for now.